neuroquery/meta_analysis.py

```diff
@@ -0,0 +1,97 @@
+"""Implement some coordinate-based meta-analysis."""
+
+import numpy as np
+import nibabel as nib
+
+from scipy.ndimage.filters import convolve
+
+
+def _uniform_kernel(r, n1=1, n2=1, n3=1):
```

Comment:
I am thinking anything related to convolutions should go in `img_utils`, WDYT?
Also, mkda and the code producing the gaussian KDE do exactly the same thing with different kernels, maybe this can be factored to remove duplicated code? the gaussian kde might benefit if you have seen that using the scipy convolution directly is faster than going through nilearn

Reply:
> I am thinking anything related to convolutions should go in img_utils, WDYT?

Yes that would be better

> Also, mkda and the code producing the gaussian KDE do exactly the same thing with different kernels, maybe this can be factored to remove duplicated code? the gaussian kde might benefit if you have seen that using the scipy convolution directly is faster than going through nilearn

Yes I will try to find a way to factor it